RestaurantManagementSystem/RestaurantManagementSystem/src/Infrastructure/Services/VerificationCodeGenerator/GenerateVerificationCode.cs

```diff
@@ -0,0 +1,16 @@
+﻿using shortid.Configuration;
+using shortid;
+
+namespace RestaurantManagementSystem.src.Infrastructure.Services.CodeGenerator
+{
+    public class GenerateVerificationCode
```

Comment:
Class name should not be like a verb, I guess it could be `CodeGenerator` or `CodeGenerationService`